incubator/astro/templates/deployment.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "astro.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "astro.name" . }}
+    helm.sh/chart: {{ include "astro.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  replicas: 1
```

Comment:
Can this run with multiple replicas?  If it can, then this should be configurable. Doesn't seem 100% necessary though

Reply:
unfortunately no, the replicas would fight each other to create/destroy monitors.

Reply:
Understandable. We'll have to work on leader election someday for this and Goldilocks